homeassistant/components/apple_tv/media_player.py

```diff
@@ -135,6 +143,19 @@ def async_device_connected(self, atv):
         # Listen to power updates
         self.atv.power.listener = self
 
+        if self.atv.features.in_state(FeatureState.Available, FeatureName.AppList):
+            self.hass.create_task(self._update_app_list())
+
+    async def _update_app_list(self):
+        _LOGGER.debug("Updating app list")
+        try:
+            apps = await self.atv.apps.app_list()
+        except Exception:  # pylint: disable=broad-except
```

Comment:
Can we avoid a broad except handler here, and only trap the expected exceptions?

Reply:
Fixed